docs/_docs/reference/other-new-features/indentation.md

```diff
@@ -502,3 +502,62 @@ indented regions where possible. When invoked with options `-rewrite -no-indent`
 The `-indent` option only works on [new-style syntax](./control-syntax.md). So to go from old-style syntax to new-style indented code one has to invoke the compiler twice, first with options `-rewrite -new-syntax`, then again with options
 `-rewrite -indent`. To go in the opposite direction, from indented code to old-style syntax, it's `-rewrite -no-indent`, followed by `-rewrite -old-syntax`.
 
+### Variant: Indentation Marker `:` for Arguments
```

Comment:
this section needs more careful reviewing because `import language.experimental.fewerBraces` has been removed, now its just part of the language

Reply:
following #16297 it seems this whole section should be left out